articles/expressroute/expressroute-locations.md

```diff
@@ -305,7 +307,7 @@ If you're remote and don't have fiber connectivity, or you want to explore other
 | **[Proximus](https://www.proximus.be/en/id_cl_explore/companies-and-public-sector/networks/corporate-networks/explore.html)**| Equinix | Amsterdam<br/>Dublin<br/>London<br/>Paris |
 | **[QSC AG](https://www2.qbeyond.de/en/)** |Interxion | Frankfurt |  
 | **[RETN](https://retn.net/products/cloud-connect)** | Equinix | Amsterdam |
-| **Rogers** | Cologix<br/>Equinix | Montreal<br/>Toronto |
+| **[Rogers]** | Cologix<br/>Equinix | Montreal<br/>Toronto |
```

Comment:
Does there need to be a URL added here?

Reply:
I just corrected a bracket that was missing, I don't work with the Rogers team.